### ISSUE TITLE
Add * and j in all the missing commands

### ISSUE THREAD

**Issue by radare** [radare/radare2#13969]
For consistency

* [ ] b*  bj
* [ ] y* yj
* [ ] cj c*
* [ ] rj r*
* [ ] P* 
* [ ] pj p* - what do here
* [ ] a* aj - whta info to show?
* [ ] uj u*
* [ ] wj w* - does it make sense?
* [ ] lj l* - json version of ls
* [ ] #!j #!*
* [ ] Lj Llj, ....
* [ ] kj have some newlnes
* [ ] k* is not working

Missing helps

* [ ] v?

**Comment by jiteshm17** [comment]
Hi, 
I am Jitesh. I am a newbie in Open Source and want to work towards solving this issue. However, I am not really able to understand the problem and how to start working on it. Can you please explain in which file of the repository should I start making changes and what exactly I should do?

**Comment by XVilka** [comment]
@jiteshm17 see the `libr/core/cmd_write.c` for `w...` commands for example, `libr/core/cmd_anal.c` for `aoq` command, see `aoj` and `ao*` on how to implement the JSON and r2 commands output, `libr/cmd.c` for more generic commands.

**Comment by ret2libc** [comment]
I'm moving this out of 4.6.0 milestone because fixing *all* commands is really an optimistic view :D

**Comment by officialcjunior** [comment]
What is the desired output for `rj`, if `r` returns, say `5240`?

**Comment by officialcjunior** [comment]
While implementing `uj`, how can I make it compatible with Windows and other OS-s?
For `u`, currently we have something like:
```
#elif __WINDOWS__
	r_cons_printf ("windows\n");
#else
	r_cons_printf ("unknown\n");
```
Does `uj` needs to cover them too? 
If yes, is there a better way to do the check?

**Comment by trufae** [comment]
Yes it should support all platforms. actually i think we shold have uname and uname -j instead of u and uj. the u command should be for undo things. also im not sure if we have tests for these commands. can u add them?

**Comment by officialcjunior** [comment]
> i think we shold have uname and uname -j instead of u and uj

Yes, it makes sense as, `u` is kind of an odd one now under `u?`, but `uname` is already a command in r2.

>  if we have tests for these commands. can u add them?

Aren't these commands system dependent? Should I really add them?
